Source/DataSources/GeoJsonDataSource.js

```diff
@@ -189,7 +189,7 @@ define([
 
             var description = properties.description;
             if (!defined(description)) {
-                entity.description = new CallbackProperty(createDescriptionCallback(properties, nameProperty), true);
+                entity.description = new CallbackProperty(createDescriptionCallback(describe, properties, nameProperty), true);
```

Comment:
Yes, that sounds more flexible - you could even gain flexibility over the `nameProperty` calculation then too. 

I could change line 192 to read something like (with appropriate checks and balances):

    entity.description = options.describeProperty(properties, nameProperty)

Then the user would define their custom `describe` function, and call `load` with `options.describeProperty` =

    function(properties, nameProperty) {
        return new CallbackProperty(createDescriptionCallback(describe, properties, nameProperty), true);
    }

Have I understood you right?